packages/types/src/session.ts

```diff
@@ -32,8 +32,10 @@ export declare namespace SessionTypes {
   }
 
   interface Updatable {
-    accounts?: Accounts;
-    namespace?: Namespace;
+    namespace?: {
```

Comment:
I mean I think we had decided on the SDK meeting that updates would require the whole namespaces

However if we want to make the update() API simpler wouldn't be easier to pass it as parameters?

update(topic, key, namespace)

Reply:
This is not related to `update` method, this is used solely in `session.find`, I think better name for this interface is `Filters`